### DISCUSSION TITLE
Embedding the entire article

### DISCUSSION THREAD

**Discussion by The-Robin-Hood** [dailydotdev/daily#649] [Feature Requests]
It would be much efficient if the entire article is embedded on the pop up page when clicked on the article. Hope you already come up with this thing, if so i would like to know which is stopping factor from this feature ?

**Comment by idoshamun** [comment]
Hi @The-Robin-Hood,

That's indeed an idea we're exploring. I shared some details and concerns in the past here: https://github.com/dailydotdev/daily/discussions/573
